procfs/example_proc_cgroup

```diff
@@ -0,0 +1,10 @@
+11:name=systemd:/user/1000.user/c1.session
```

Comment:
Why do we need this file here?

Reply:
it was in original sources for a reason I guess. Here is where it was added:
```
commit 6ef3de1d5f1d4b19c154ba15f2bd4686b6036f0c
Author: Ananya Kumar <skywalker94@gmail.com>
Date:   Mon Aug 3 17:28:33 2015 -0700

    Add QoS support on node
```